modules/objdetect/src/face_detect.cpp

```diff
@@ -283,7 +287,11 @@ Ptr<FaceDetectorYN> FaceDetectorYN::create(const String& model,
                                            const int backend_id,
                                            const int target_id)
 {
+    #ifdef HAVE_OPENCV_DNN
     return makePtr<FaceDetectorYNImpl>(model, config, input_size, score_threshold, nms_threshold, top_k, backend_id, target_id);
+    #else
+    CV_Error(cv::Error::StsNotImplemented, "cv::FaceDetectorYN is depend on dnn module.");
```

Comment:
> is depend on

just "depends on" ?

or "requires enabled 'dnn' module"

Reply:
thanks, I'll fix it.